frb/tests/test_galfit.py

```diff
@@ -16,8 +18,8 @@
 from frb.frb import FRB
 
 from frb.galaxies import galfit as glf
-remote_data = pytest.mark.skipif(os.getenv('FRB_GDB') is None,
-                                        reason='test requires dev suite')
+galfit_exec = pytest.mark.skipif(find_executable('galfit') is None,
```

Comment:
Is it possible to force tests? I'm not very familiar with `pytest` but that seems like a useful option to have.

Reply:
what do you mean by "force tests" ?

Reply:
Nevermind, I think I wasn't sure how skipif works. Now that I look at my comment, it looks silly.